Applications/WebPortal/WebApp/library/source/pages/account_directory_page/local_account_directory_model.ts

```diff
@@ -47,6 +47,27 @@ export class LocalAccountDirectoryModel extends AccountDirectoryModel {
         resolve(this._accounts.get(group).slice());}, 100);
       });
   }
+  public async loadFilteredAccounts(
+    filter: string): Promise<Beam.Map<Beam.DirectoryEntry, AccountEntry[]>> {
+    if(!this.isLoaded) {
+      throw Error('Model not loaded.');
+    }
+    return new Promise<Beam.Map<Beam.DirectoryEntry, AccountEntry[]>>(
+      (resolve) => {
+      setTimeout(() => {
+        const map = new Beam.Map<Beam.DirectoryEntry, AccountEntry[]>();
+        for(const group of this._groups) {
+          const accounts: AccountEntry[] = [];
+          for(const account of this._accounts.get(group)) {
+            if(account.account.name.includes(filter)) {
```

Comment:
Filters are done strictly by prefix.

Reply:
What do you mean by that?

Reply:
Where does the spec say otherwise? Could be worth correcting the spec.